### DISCUSSION TITLE
[Question] Starting app

### DISCUSSION THREAD

**Discussion by n-ext** [AlbericoD/overwolf-modern-react-boilerplate#21] [Q&A] [answered]
I'm having trouble with starting app generated using this template in Overwolf.

I don't understand which command should I use to create (unpacked) extension. Could you provide short instruction how to do it?

**Comment by AlbericoD** [comment, marked as answer]
Sure


1° Run
```sh
$ npx create-react-app my-awesome-overlay-app --template overwolf-typescript-redux
```

2° Compile
```sh
$ npm run build
# or
$ yarn run build
```

3° Organize Folder
```sh
$ npm run pack-overwolf
# or
$ yarn run pack-overwolf
```

4° Load 'upacked' app

- Open Overwolf App Store, then Click on Development options
![gh-1](https://user-images.githubusercontent.com/25888798/137593629-e2a99172-2820-4036-8b4d-df8322f791a8.PNG)


- Click on **Load upacked extension...** button.
![gh-2-c](https://user-images.githubusercontent.com/25888798/137594103-07bd97e8-2243-47f5-b379-e1b272f0d955.png)



- Select the build folder, then click in 'app' folder
![gh03](https://user-images.githubusercontent.com/25888798/137593810-3f1d81f2-6918-4fbb-8eac-802303072b51.PNG)

- You can see your app installed
![ihhh](https://user-images.githubusercontent.com/25888798/137594067-ded065d5-4546-4076-9c28-2a100eb4a6cc.jpg)

![gh-5](https://user-images.githubusercontent.com/25888798/137593841-95c6e147-f83c-4057-b8b0-37198f23fb3e.PNG)


I hope this helps you understand, this process is also available in the overwolf documentation in [Install the app as "unpacked extension"](https://overwolf.github.io/docs/start/sample-app-overview#5-install-the-app-as-unpacked-extension)
